rktlet/runtime/helpers.go

```diff
@@ -266,7 +267,15 @@ func generateAppAddCommand(req *runtimeApi.CreateContainerRequest, imageID strin
 	}
 	// TODO(yifan): logpath
 	// TODO(yifan): privileged
-	// TODO(yifan): mount, volumes
+
+	for _, mnt := range config.GetMounts() {
+		if mnt == nil {
+			glog.Warningf("unexpected nil mount: %v, %+v", mnt, config)
+			continue
+		}
+		volumeName := uuid.NewUUID()
+		cmd = append(cmd, fmt.Sprintf("--mnt-volume=name=%s,kind=host,source=%s,target=%s,readOnly=%t", volumeName, mnt.GetHostPath(), mnt.GetContainerPath(), mnt.GetReadonly()))
```

Comment:
Since that's not expressed in the k8s api, we'd just have to pick a default. Since it already defaults to recursive, and that's the behaviour I'd expect, I don't see a reason to add it explicitly here.